### ISSUE TITLE
Contributing guideline

### ISSUE THREAD

**Issue by thuan1412** [python-lsp/python-lsp-server#216]
Hi, thank you for making this project. I'm using it and it's really great.
I want to understand the project and maybe contribute to this project. However, I feel it's difficult to understand the project, run from the source

Could we have a `CONTRIBUTING.md` file that describes the project and how to contribute?

**Comment by staticf0x** [comment]
I would also be interested in this

**Comment by ccordoba12** [comment]
Hey guys, I'm the maintainer of this project, but I also maintain several other projects, so I don't have time for this, sorry.

Perhaps @doolio could be interested on this?

**Comment by doolio** [comment]
Thanks Carlos but I'm not sure I'm best placed to do so. At least not yet. I'm only learning to programme myself and started with python. As I develop my skills and use `pylsp` I can look to understand how it works. Perhaps then I can make such a guide if one still does not exist.

**Comment by ccordoba12** [comment]
No problem @doolio, that's fine.

**Comment by staticf0x** [comment]
We could try to put something together, I tried to dig in the code yesterday and got the `develop` branch to play nice with my sublime-text, so maaaybe I could try once I get a better hang of it.

**Comment by ccordoba12** [comment]
Yeah, that's a very good idea @staticf0x! If you or someone else comes up with a Contributing guide in a pull request, I'll provide guidance with suggestions and improvements to merge it.
